src/main/java/spireMapOverhaul/BetterMapGenerator.java

```diff
@@ -70,6 +70,7 @@ public ArrayList<ArrayList<MapRoomNode>> generate(Random rng, int width, int hei
             float zoneRate = 1;
 
             for (AbstractZone queuedZone : queueCommandZones) {
+                mapGenLogger.info(queuedZone.id);
```

Comment:
No need to keep this in. Changes to how `addzone` works means that zones added with it will more reliably show up.